airflow/jobs/scheduler_job_runner.py

```diff
@@ -86,6 +86,10 @@
 DR = DagRun
 DM = DagModel
 
+# type alias for task concurrency map,
+# (dag_id, run_id, task_group_id) -> concurrency limitation and set of running map indexes
+TaskGroupConcurrencyMap = Dict[Tuple[str, str, str], Tuple[int, Set[int]]]
```

Comment:
Is the concurrency limit (first int in the value) used anywhere? I tried but could not find one. It seems the code can be simplified a bit if we don’t pass that value.

Reply:
Yes, you are right.
This field can be removed.